dev-preview/README.md

```diff
@@ -97,6 +97,48 @@ Installation instructions can be found in the [repo](https://github.com/flightct
 
 Instructions for managing fleets can be found [here](https://github.com/flightctl/flightctl/blob/main/docs/user/managing-fleets.md) while for managing devices are available [here](https://github.com/flightctl/flightctl/blob/main/docs/user/managing-devices.md). One of the benefits of using Edge Management together with ACM is the ability to enroll devices running Red Hat Device Edge seamlessly.
 
+## Openshift Cluster API Operator
```

Comment:
Let's mention:
ROSA Hosted Control Planes in the title or at least this entry paragraph near the start.